services/node_create.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package com.hedera.hapi.node.addressbook;
+package proto;
```

Comment:
Please do not move these to the `proto` package.
We will not use `proto` package going forward (we never should have used it at all, as it's an example namespace not to be used in production code).

Reply:
The reason I had to change the package name is:
 **Key** will be invalid type if using package com.hedera.hapi.node.addressbook since it's defined in basic_types.proto and under package proto. 
 We might need to change all the package names later together.

Reply:
That's not accurate.  `Key` is entirely usable, it's just referenced as `proto.Key` because of the bad package in `basic_types`.
There are several similar examples already in this file.

Reply:
Updated.